<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>624</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>809</width>
    <height>556</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Все тайны факультета УИТС</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource_form.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Программа тестирования знаний истории факультета УИТС.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="verticalFrame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
     padding: 2px;
     margin:20px;
	border-radius: 2px;
	background-color: rgb(255, 255, 255);
 }</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="font">
             <font>
              <family>Franklin Gothic Book</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 0px solid gray;
margin: 0px;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Franklin Gothic Book'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt;&quot;&gt;dfbdfbdfbdfbdfbdfbdfbdfb&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt;&quot;&gt;dfvdbdfbdf&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt;&quot;&gt;fbdfbdfbdfbd&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt;&quot;&gt;srbe e5g v eg df&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_2">
              <property name="font">
               <font>
                <family>Franklin Gothic Book</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
     border: 1px solid grey;
     border-radius: 1px;
     background-color:rgb(123, 206, 152);
     min-width: 80px;
	min-height:20px;
 }

 QPushButton:pressed {
     background-color: rgb(108, 180, 132);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="font">
               <font>
                <family>Franklin Gothic Book</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
     border: 1px solid grey;
     border-radius: 1px;
     background-color:rgb(123, 206, 152);
     min-width: 80px;
	min-height:20px;
 }

 QPushButton:pressed {
     background-color: rgb(108, 180, 132);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_4">
              <property name="font">
               <font>
                <family>Franklin Gothic Book</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
     border: 1px solid grey;
     border-radius: 1px;
     background-color:rgb(123, 206, 152);
     min-width: 80px;
	min-height:20px;
 }

 QPushButton:pressed {
     background-color: rgb(108, 180, 132);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_3">
              <property name="font">
               <font>
                <family>Franklin Gothic Book</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
     border: 1px solid grey;
     border-radius: 1px;
     background-color:rgb(123, 206, 152);
     min-width: 80px;
	min-height:20px;
 }

 QPushButton:pressed {
     background-color: rgb(108, 180, 132);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="styleSheet">
             <string notr="true">QProgressBar::chunk {
     background-color: #7bce98;
     width: 20px;
 }
 QProgressBar {
     border: 1px solid grey;
     border-radius: 1px;
     text-align: center;
 }</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="verticalFrame">
         <property name="styleSheet">
          <string notr="true">QFrame {
    padding: 2px;
    margin:0 20px;
	border-radius: 2px;
	background-color: rgb(255, 255, 255);
 }</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTextBrowser" name="textBrowser_2">
            <property name="font">
             <font>
              <family>Franklin Gothic Book</family>
              <pointsize>10</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"> padding: 1px;
 margin:1px;</string>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Franklin Gothic Book'; font-size:10pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;sdasc&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">image: url(:/vsuet);
min-height:170px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="formFrame">
            <layout class="QFormLayout" name="formLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="1" column="0">
              <widget class="QLineEdit" name="lineEdit">
               <property name="font">
                <font>
                 <family>Franklin Gothic Book</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"> QLineEdit {
     border: 0px solid gray;
     border-radius: 0px;
     padding: 0 8px;
	 color: rgb(0,0,0);
     selection-background-color: darkgray;
 }</string>
               </property>
               <property name="text">
                <string>Дано ответов:</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="font">
                <font>
                 <family>Franklin Gothic Book</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="styleSheet">
                <string notr="true"> QLineEdit {
     border: 0px solid gray;
     border-radius: 0px;
     padding: 0 8px;
	 color: rgb(0,0,0);
     selection-background-color: darkgray;
 }</string>
               </property>
               <property name="text">
                <string>True:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLineEdit" name="lineEdit_3">
               <property name="font">
                <font>
                 <family>Franklin Gothic Book</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="styleSheet">
                <string notr="true"> QLineEdit {
     border: 0px solid gray;
     border-radius: 0px;
     padding: 0 8px;
	 color: rgb(0,0,0);
     selection-background-color: darkgray;
 }</string>
               </property>
               <property name="text">
                <string>False:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <family>Franklin Gothic Book</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>Franklin Gothic Book</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <family>Franklin Gothic Book</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="styleSheet">
               <string notr="true">QPushButton {
     border: 1px solid grey;
     border-radius: 1px;
     background-color:rgb(209, 95, 71);
     min-width: 60px;
	min-height:20px;
 }

 QPushButton:pressed {
     background-color: rgb(108, 180, 132);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
              </property>
              <property name="text">
               <string>В полный экран!</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="styleSheet">
               <string notr="true">QPushButton {
     border: 1px solid grey;
     border-radius: 1px;
     background-color:rgb(123, 206, 152);
     min-width: 60px;
	min-height:20px;
 }

 QPushButton:pressed {
     background-color: rgb(108, 180, 132);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
              </property>
              <property name="text">
               <string>Показать авторов</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/gr3);
vertical-align: bottom;
min-height:100px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource_form.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
